docs/HOWTO_AZURE_DEPLOY.md

```diff
@@ -0,0 +1,149 @@
+- Set up nginx
+- Go into nginx to configure the URL properly routes to the application when you go to it.
+- Link: All the steps for starting with a completely blank and empty server.
+- Go to section 4 to set up nginx
+
+
+## Set up the SSL
+- Most organizations have their own certificates.
+- But in case you don’t: Visit certbot.eff.org
+- Follow the instructions on the site
+- It assumes you’ve already set up your EngineX.
+- You’ll have to type in your contact information
+- Then it’s all up and running
+
+## Install Redis
+- Something set up by default within the Docker container.  It sets up a Redis instance.
+- We’re going to deploy redis separately within the server.
+- Since we’ve already broken the database option out of the Docker File.
+- But need to look for [Azure Cache for Redis instructions] (https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_CONNECT_WITH_REDIS.md)
+
+## Install Postgres
+- In our documentation there are some information for setting this up.
+- The way we set it up is that we have a different database set in Azure and then connected within the container of your Azure project.
+- More [expanded Docker descriptions](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_USE_POSTGRESQL.md) if people who are doing this wanted to go that route
+- Configure environment variables
+- Start sticking in the variables you actually want in the environment variable file.
+    - SUPRESS_SELF_INVITE=1
+    - DB_TYPE=pg
+    - DEFAULT_SERVICE=twilio
+- If you are adding the variables at an organization level through the front-end, then these variables do not need to be set in the .env file
+    - TWILIO_ACCOUNT_SID=
+    - TWILIO_MESSAGE_SERVICE_SID
+    - TWILIO_MESSAGE_SERVICE_SID
+- If you’re doing a smaller organization with one campaign, you don’t want to set this variable to 1.  But if you are working with multiple orgs, setting this variable to 1 allows you to override Twilio in whatever is in the Global Variable file.
+    - TWILIO_MULTI_ORG=1
+        - Set this variable, because it is not automatically generated for backwards compatibility.
+    - TWILIO_VALIDATION=1
+- Variables to remove from .env file to ensure Spoke successfully processes responses from the text:
+    - TWILIO_AUTH_TOKEN=[ADD TOKEN HERE]
+    - TWILIO_MESSAGE_CALLBACK_URL
+- Call Back URL needs to be updated.
+- There might be a problem with the callback not working due to an Azure firewall
+- Usually there is an IP White List for the different access for things
+- It’s probably doing something wrong with the environment variables.
+
+
+## Heroku Deployment
+- In order for people to download csv’s we used Bucketeer
+- Where the csv files go if you are trying to export data out of spoke.
+- Designated temp directory for csv exports
+- What is the expected workflow is for the data exports
+    - MAILGUN_API_KEY, knows that it works but doesn’t know what’s going on.
+    - USING UBUNTU TO DEPLOY THIS: Email is included as a thing works.
```

Comment:
These lines sound confusing to me

Reply:
That's because it was scratch notes from a conversation we were having.  I've updated this section to be more outward facing.